<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QDialog" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sPS - Shadow Port Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/app-icon.png</normaloff>:/icons/app-icon.png</iconset>
  </property>
  <widget class="QLabel" name="lblHost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Host/IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPortBegin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHost">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBeginPort">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPortEnd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ending Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEndPort">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1024</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScan">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>300</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/button-icon.png</normaloff>:/icons/button-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
